### PR TITLE
rspec-yank-last-command

### PR DESCRIPTION
**Pull request by Kukunin**

### CODE REVIEW

**Comment by Kukunin** [comment]
closes #158

File: rspec-mode.el [hunk 1]
```diff
@@ -781,12 +791,18 @@ or a cons (FILE . LINE), to run one example."
...
-                                     ,(rspec-runner-options opts)
-                                     ,target) " ")))
-             'rspec-compilation-mode)))
+    (compile
+     (rspec-compile-command target opts)
+     'rspec-compilation-mode)))
+
+(defun rspec-compile-command (target &optional opts)
+  "Composes RSpec command line for the compile function"
+  (let ((default-directory (or (rspec-project-root) default-directory)))
```
Comment (dgutov) [inline rspec-mode.el]: I don't like this construct already, and here you are adding a second place it's used. Even though one caller of this new function is the other place.

Can we do it in some different way?

Comment (Kukunin) [inline rspec-mode.el]: We can do. This is my first PR on elisp, so I tried to not move/refactor a lot of things.

Do you have ideas about how to refactor this? extract to another function?

Comment (dgutov) [inline rspec-mode.el]: What if you just drop this `let`?

Comment (Kukunin) [inline rspec-mode.el]: ahh.. I see - I duplicated this let. My fault

**Comment by dgutov** [comment]
Thanks!

**Pull request merged**